### ISSUE TITLE
Task-017: Task administrator

### ISSUE THREAD

**Issue by Supremaciu** [Supremaciu/G98-DP2#55]
As soon as the assignees that are in charge of implementing task for anonymous users have finished, this assignee will have to replicate the code into the administrator users. Optionally, the assignee can implement the corresponding restrictions. This will eventually be done in another task in the future, which is why it is currently optional

Tasks that must be done before this one: #61
